src/selinux/cockpit.te

```diff
@@ -87,8 +91,13 @@ auth_write_login_records(cockpit_session_t);
 allow cockpit_session_t init_t:unix_stream_socket sendto;
 
 # cockpit-session does these when authenticating via GSSAPI
+# also when starting a user agent
 allow cockpit_session_t tmp_t:file { create write open unlink };
 
+# TODO: I THINK THIS IS WRONG
+# cockpit-session can spawn a ssh-agent
+allow unconfined_t ssh_agent_exec_t:file entrypoint;
```

Comment:
Yup, this looks wrong. Should be ```cockpit_session_t```. But perhaps this is because of some interaction with ```pam_selinux.so``` that you have to use ```unconfined_t``` here?